src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js

```diff
@@ -1,5 +1,7 @@
 import {normalTextMixin} from './lib.js'
 
+// TODO(matthew): Can we get rid of this????
```

Comment:
I don't want to get rid of it yet, i personally like the ``:`` syntax and think we may want to change to it eventually.